src/main/java/com/synopsys/integration/jenkins/scan/global/ApplicationConstants.java

```diff
@@ -73,4 +74,6 @@ public class ApplicationConstants {
     public static final String COVERITY_INPUT_JSON_PREFIX = "coverity_input";
     public static final String POLARIS_INPUT_JSON_PREFIX = "polaris_input";
 
+    public static final String BRIDGE_NETWORK_AIRGAP_KEY = "BRIDGE_NETWORK_AIRGAP";
```

Comment:
So need to check whether we should make this lowercase like other parameters.

Reply:
Done.